content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,744 @@
+to generate and push a ca certificate secrets to both the `CLUSTER1` and `CLUSTER2`
+clusters.
+
+{{< tip >}}
+If you currently have a single cluster with a self-signed CA (as described
+in [Getting Started](/docs/setup/getting-started/)), you need to
+change the CA using one of the methods described in
+[certificate management](/docs/tasks/security/cert-management/). Changing the
+CA typically requires reinstalling Istio. The installation instructions
+below may have to be altered based on your choice of CA.
+{{< /tip >}}
+
+## Install Istio
+
+The steps for installing Istio on multiple clusters depend on your
+requirements for network and control plane topology. This section
+illustrates the options for two clusters. Meshes that span many clusters may
+employ more than one of these options. See
+[deployment models](/docs/ops/deployment/deployment-models) for more
+information.
+
+{{< tabset category-name="install-istio" >}}
+
+{{< tab name="Multi-Primary" category-value="multi-primary" >}}
+
+The steps that follow will install the Istio control plane on both `CLUSTER1` and
+`CLUSTER2`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. Both
+clusters reside on the `NETWORK1` network, meaning there is direct
+connectivity between the pods in both clusters.
+
+Each control plane observes the API Servers in both clusters for endpoints.
+
+Service workloads communicate directly (pod-to-pod) across cluster boundaries.
+
+{{< image width="75%"
+    link="multi-primary.svg"
+    caption="Multiple primary clusters on the same network"
+    >}}
+
+**Configure `CLUSTER1` as a primary**
+
+Install Istio on `CLUSTER1`:
+
+{{< text bash >}}
+$ cat <<EOF > ./cluster1.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: MESH1
+      multiCluster:
+        clusterName: CLUSTER1
+      network: NETWORK1
+      meshNetworks:
+        NETWORK1:
+          endpoints:
+          - fromRegistry: CLUSTER1
+          - fromRegistry: CLUSTER2
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+$ istioctl install --context=${CTX_CLUSTER1} -f cluster1.yaml
+{{< /text >}}
+
+**Configure `CLUSTER2` as a primary**
+
+Install Istio on `CLUSTER2`:
+
+{{< text bash >}}
+$ cat <<EOF > ./cluster2.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: MESH1
+      multiCluster:
+        clusterName: CLUSTER2
+      network: NETWORK1
+      meshNetworks:
+        NETWORK1:
+          endpoints:
+          - fromRegistry: CLUSTER2
+          - fromRegistry: CLUSTER1
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+$ istioctl install --context=${CTX_CLUSTER2} -f cluster2.yaml
+{{< /text >}}
+
```

Comment:
For multi primary, the diagram didn't show gateways while the plan showed gateways.  This is a bit confusing when I reviewed this tab.

Reply:
Are you referring to the presence of `istio-eastwestgateway` in the `IstioOperator` yaml? Thing is that the gateway isn't needed for this case, so we don't actually deploy it. Not sure if it's valid to leave out `gateways` altogether, but if that works maybe that's a better solution here.

@stevenctl WDYT?

Reply:
I assume the gateways aren't used for workloads to communicate if so, it is confusing we also call it eastwestgateway.

Reply:
I do agree that we don't need to show the gateways in the multi-primary guide, they're not really necessary. 

@linsun 
The eastwest gateways are used for:
- multinetwork workload traffic
- discovery from outside the clsuter

eastwest sort of implies it's mostly used for workload traffic, so I can see how that name is confusing.

Reply:
@linsun east-west is a pretty standard term, but I'm open to other names if they're more clear.

Reply:
If removing the gateways are okay for this example, I'd vote for removing it because it is quite confusing to have it :) @stevenctl @nmittler

Reply:
@nmittler the name east-west is good.  It is just confusing here when it is not used for workload traffic.